test/strongbox/core_test.clj

```diff
@@ -216,9 +216,11 @@
           every-addon-api (slurp (fixture-path "curseforge-api-addon--everyaddon.json"))
           every-other-addon-api (slurp (fixture-path "curseforge-api-addon--everyotheraddon.json"))
 
+          ;; todo: this dummy catalogue munging is getting out of hand
```

Comment:
replace with a v2 json fixture. no need to preserve backwards compatibility or anything like that here. read-catalogue/v1-coercer will be getting it's own tests to ensure backwards compability.